osu.Game.Rulesets.Catch/UI/CatcherTrail.cs

```diff
@@ -34,11 +29,40 @@ public CatcherTrail()
             };
         }
 
-        protected override void FreeAfterUse()
+        protected override void OnApply(CatcherTrailEntry entry)
         {
-            ClearTransforms();
+            Scale = entry.Scale;
+            Position = new Vector2(entry.Position, 0);
             Alpha = 1;
```

Comment:
Other places do this by doing `ApplyTransformsAt(double.MinValue)` in `OnFree()` before clearing transforms, but I suppose this also works.